README.md

```diff
@@ -2,11 +2,19 @@
 
 ---
 
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-52-orange.svg?style=flat-square)](#contributors-)
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
-
-[![Netlify Status](https://api.netlify.com/api/v1/badges/b2137407-b765-46c4-95b5-a72d9b1592ab/deploy-status)](https://app.netlify.com/sites/asyncapi-website/deploys)
+<br/>
+<p align="center">
+<a href="https://github.com/asyncapi/website" alt="AsyncAPI GitHub website contributors">
+<img src="https://img.shields.io/github/contributors/asyncapi/website.svg?color=orange" /></a>
+<a href="https://github.com/asyncapi/website/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22" alt="Good First AsyncAPI issue">
+<img src="https://img.shields.io/github/issues/asyncapi/website/good%20first%20issue.svg?color=%23DDDD00" /></a>
+<a href="https://asyncapi.slack.com/" alt="AsyncAPI Slack">
+<img src="https://img.shields.io/badge/Slack-@website.svg?logo=slack" /></a>
```

Comment:
A suggestion: We can add `AsyncAPI` in this badge, it is odd to have a single badge of other type than others in list.

Reply:
are you talking about Slack?

Reply:
Yes

Reply:
I think we can add `AsyncAPI slack`, wdyt?